src/rules/delta/extended/marginals.jl

```diff
@@ -0,0 +1,37 @@
+export marginalrule
```

Comment:
Might be good to indicate that these and other updates are directly taken from ForneyLab. Not all implementations might follow ReactiveMP conventions; in any case they are not optimized for computation, but make-it-fast may come later.

Reply:
Added comments